tests/cephfs/snapshot_clone/snap_schedule_retention_vol_subvol.py

```diff
@@ -826,6 +836,69 @@ def snap_sched_multi_fs(snap_test_params):
     return post_test_params
 
 
+def snap_sched_non_existing_path(snap_test_params):
+    """Verify Snapshot schedule can be created for non-existing path.
+    After the start-time is hit for the
+    schedule, the snap-schedule module should have deactivated the schedule without throwing a traceback error.
+    The snap-scheule module should continue to remain responsive and a log message should be seen in the logs
+    about deactivating the schedule."""
+    client = snap_test_params["client"]
+    snap_util = snap_test_params["snap_util"]
+    fs_util = snap_test_params["fs_util"]
+    snap_util.enable_snap_schedule(client)
+    snap_test_params["validate"] = True
+    snap_test_params["path"] = "/non-existent"
+    sched_list = ["1M"]
+    snap_test_params["retention"] = "3M"
+    snap_test_params["start_time"] = get_iso_time(client)
+    snap_test_params["sched"] = sched_list[0]
+    installer_node = snap_test_params["installer_node"]
+    post_test_params = {}
+    test_fail = 0
+    out, rc = client.exec_command(sudo=True, cmd="ceph config set mgr log_to_file true")
+    log.info("Create snap-schedule for non-existing path")
+    snap_util.create_snap_schedule(snap_test_params)
+    log.info("Verify schedule for non-existent path got deactivated")
+    if snap_util.check_snap_sched_active(client, snap_test_params["path"], "False"):
+        test_fail = 1
+
+    log.info("Verify message in mgr log for non-existing path")
+    out, _ = installer_node.exec_command(sudo=True, cmd="cephadm ls")
+    data = json.loads(out)
+    fsid = data[0]["fsid"]
+    cmd = f"cd /var/log/ceph/{fsid}/;ls *mgr*log"
+    out, _ = installer_node.exec_command(sudo=True, cmd=cmd)
```

Comment:
Their might be a situation were the installer node might not be the active mgr. Can we consider getting the active mgr and verify the logs if possible. May be as enhancement